src/goto-programs/rw_set.cpp

```diff
@@ -49,6 +49,37 @@ void rw_sett::read_write_rec(
       {
         return; // ignore for now
       }
+
+      //improvements for CUDA features
```

Comment:
Can you add a TODO here? We should investigate in another PR why our data race check is broken.